tests/certification/pubsub/rabbitmq/mtls_sasl_external/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: "3.7"
+services:
+  rabbitmq_external_sasl:
+    image: rabbitmq_external_sasl
+    build: mtls_sasl_external/docker_sasl_external
```

Comment:
Is this perhaps a relative path that needs to be updated?

Reply:
I assume so much, yes. Thanks. I don't think I've ever pruned all images. I did it now, and it works with ways, but maybe docker desktop compose might have a different path lookup scheme....